backend/README.md

```diff
@@ -1,75 +1,109 @@
-## Installation
+# Backend for Learning Journey Planning System
 
-Navigate to root folder of `backend`
+This is the application root for the backend system of LJPS.
 
-```
+## Prerequisites
+
+- Recommended platform
+    - Development: macOS Monterey (Intel)
+    - Production: Linux (Ubuntu 16.04)
+- Python 3.8
+- MySQL >=8.0 server
+
+### Set-up Local Directories
+
+Clone this repository or download the files to local directory.
+Open a terminal session and navigate to this application root (`.../is212g5t7-project/backend`)
+
+```bash
 cd /path/to/is212g5t7-project/backend
 ```
 
-Install dependencies
+### Provide Environment Variables
 
-```
-pip install --upgrade pip
-pip install -r requirements.txt
-```
+This backend application relies on the accompanying MySQL server for data persistence. We need to provide it the following information. Edit the `.env.example` using any text editor (`vi .env.example`).
+    1. Replace `<>` fields with the respective information
+    2. Rename `.env.example` to `.env`
```

Comment:
Not sure why but the two steps are not shown as an ordered list in the image, if you meant to do that.